pkgs/by-name/fr/friction-graphics/package.nix

```diff
@@ -0,0 +1,92 @@
+{ lib
+, clangStdenv
+, cmake
+, expat
+, fetchgit
+, ffmpeg_4-full
+, fontconfig
+, freetype
+, glib
+, gn
+, harfbuzzFull
+, icu
+, libGL
+, libjpeg_turbo
+, libpng
+, libsForQt5
+, libunwind
+, libwebp
+, ninja
+, pcre2
+, pkg-config
+, python3
+, qt5Full
+, xorg
+, zlib
+}:
+
+clangStdenv.mkDerivation rec {
+  pname = "friction.graphics";
+  version = "0.9.6";
+  src = fetchgit {
+    url = "https://github.com/friction2d/friction";
+    rev = version;
+    sha256 = "sha256-QjwkijsTAEEN1VqGtIfsNsHTM1DIRUp+hfABuXoj+nU=";
+    fetchSubmodules = true;
+  };
+  buildInputs = [
+    expat
+    ffmpeg_4-full
+    fontconfig
+    freetype
+    glib
+    gn
+    harfbuzzFull
+    icu
+    libGL
+    libjpeg_turbo
+    libpng
+    libsForQt5.qscintilla
+    libunwind
+    libwebp
+    ninja
+    pcre2
+    python3
+    qt5Full
+    xorg.libX11
+    zlib
+  ];
+  nativeBuildInputs = [ cmake pkg-config ];
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DCMAKE_INSTALL_PREFIX=bin"
+    "-DHARFBUZZ_INCLUDE_DIRS=${harfbuzzFull.dev}/include"
+    "-DQSCINTILLA_INCLUDE_DIRS=${libsForQt5.qscintilla}/include"
+    "-DQSCINTILLA_LIBRARIES_DIRS=${libsForQt5.qscintilla}/lib/"
+    "-DQSCINTILLA_LIBRARIES=libqscintilla2.so"
+    "-G Ninja"
+  ];
+  patchPhase = ''
+    sed -i 's|''${SKIA_SRC}/bin/gn)|${gn}/bin/gn)|' src/engine/CMakeLists.txt
+    sed -i 's|fontconfig|${fontconfig.lib}/lib/libfontconfig.so|' src/cmake/friction-common.cmake
+    grep -rl 'hb' | xargs sed -Ei 's/(["<])(hb.*\.h)/\1harfbuzz\/\2/'
+  '';
+  buildPhase = ''
+    export VERBOSE=1
+    cmake --build . --config Release
```

Comment:
```suggestion
    cmake --build . --config Release
```
Please use cmakeFlags to pass --config Release if necessary

Reply:
I'm not sure if that's possible, this is what I get when I try to do that:
![2024-07-20-011255_hyprshot](https://github.com/user-attachments/assets/2956fedc-fc41-4c68-8b9d-d55ca834f461)

It's a flag that's [explicitly documented in their README](https://github.com/friction2d/friction?tab=readme-ov-file#build-on-linux) so I assumed it was necessary